src/app.py

```diff
@@ -5,7 +5,8 @@
 import pandas as pd
 
 df = pd.read_csv("../data/clean/movies_clean_df.csv", index_col=0).rename(
-    columns={'Major Genre': 'Major_genre', 'Year': 'Year'}, inplace=False)
+    columns={'Major Genre': 'Major_genre', 'Year': 'Year', 
```

Comment:
```suggestion
    columns={'Major Genre': 'Major_genre',  
```
is it okay to get rid of the 'Year':'Year'?